test/e2e/node/pod_resize.go

```diff
@@ -276,10 +272,6 @@ func doPodResizeTest() {
 	var podClient *framework.PodClient
 	var ns string
 
-	if !utilfeature.DefaultFeatureGate.Enabled(InPlacePodVerticalScalingFeature) {
```

Comment:
@vinaykul @ehashman, please correct me if I am wrong. So I made the e2e testing changes according to our previous discussion.
 `so we don't call SIGDescribe anywhere in the e2e tests that are defined.  this is mandatory, they have to be tagged by owner SIG. furthermore we don't have any selectors defined that will indicate whether or not the tests should run. typically we do this with a [Feature:Foo] tag on the test name. then we modify the GCP alpha job, and run that, we don't necessarily need to make a special node one.`

Reply:
And the PR we have to modify the GCP alpha job is here [test-infra#23177](https://github.com/kubernetes/test-infra/pull/23177)

Reply:
@ehashman, could you help review both?